locale/circuitpython.pot

```diff
@@ -2642,7 +2642,7 @@ msgstr ""
 msgid "can't set 512 block size"
 msgstr ""
 
-#: py/objnamedtuple.c
+#: py/objexcept.c py/objnamedtuple.c
```

Comment:
Your PR's have these changes in `circuitpython.pot`. Could you merge the latest `main` from upstream to make yours match? Otherwise we will have churn in this file.

Reply:
Also a change here causes everything to be rebuilt on each PR push, instead of just the board build.